build.gradle

```diff
@@ -196,9 +203,12 @@ def Task getPublishArtifactsTask(String buildVersion, Project subproject) {
 allprojects {
     apply plugin: 'com.bmuschko.docker-remote-api'
 
-    task copyDocker(type: Sync) {
-        from "${project.projectDir}/Dockerfile"
-        into "build/docker/"
+    // check if the copyDocker task already exists as it may have already been added by cloud
+    if (tasks.findByName('copyDocker') == null) {
```

Comment:
Groovy nit:  you can just do `!tasks.findByName(...)` without the `== null` bit.